ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -241,6 +263,7 @@ module.exports = function OpenlayersMap(
       map.getView().fit(extent, {
         size: map.getSize(),
         duration: 500,
+        ...opts,
```

Comment:
What kind of opts get passed through?

Reply:
This allows the caller to override animation time and other values.